app/src/main/java/com/projectit3048c/MainViewModel.kt

```diff
@@ -22,7 +22,9 @@ class MainViewModel(var foodService : IFoodService =  FoodService()) : ViewModel
     var selectedFoodAmount by mutableStateOf(FoodAmount())
     val NEW_FOODAMOUNT = "New Food"
     var user: User? = null
-    private lateinit var firestore : FirebaseFirestore
+    private var firestore: FirebaseFirestore = FirebaseFirestore.getInstance().apply {
+        firestoreSettings = FirebaseFirestoreSettings.Builder().build()
+    }
```

Comment:
I'd lean towards leaving this as it were. firestore is initialized in the init{} method, which is a bit more readable than initializing it where it is declared.  Plus, that initialization in init {} was not removed as part of this change, so it effectively happens twice now.